mcs/resource_mcs_db_user.go

```diff
@@ -284,23 +284,40 @@ func resourceDatabaseUserUpdate(d *schema.ResourceData, meta interface{}) error
 		}
 	}
 	var userUpdateParams userUpdateOpts
+
+	if d.HasChange("password") {
+		_, new := d.GetChange("password")
+		userUpdateParams.User.Password = new.(string)
+		stateConf := &resource.StateChangeConf{
+			Pending:    []string{"BUILD"},
+			Target:     []string{"ACTIVE"},
+			Refresh:    databaseUserStateRefreshFunc(DatabaseV1Client, dbmsID, userName, dbmsType),
+			Timeout:    d.Timeout(schema.TimeoutCreate),
+			Delay:      dbUserDelay,
+			MinTimeout: dbUserMinTimeout,
+		}
+		err = userUpdate(DatabaseV1Client, dbmsID, userName, &userUpdateParams, dbmsType).ExtractErr()
+		if err != nil {
+			return fmt.Errorf("error updating mcs_db_user: %s", err)
+		}
+		_, err = stateConf.WaitForState()
```

Comment:
i'm not sure we need to wait for changing password. it's a synchronous operation